engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/ExecutionStore.scala

```diff
@@ -22,19 +21,23 @@ object ExecutionStore {
       case _ => None
     }
 
-    new ExecutionStore(keys.flatten.map(_ -> NotStarted).toMap)
+    new ExecutionStore(keys.flatten.map(_ -> NotStarted).toMap, keys.nonEmpty)
   }
 }
 
-case class ExecutionStore(private val store: Map[JobKey, ExecutionStatus]) {
+final case class ExecutionStore(private val statusStore: Map[JobKey, ExecutionStatus], hasNewRunnables: Boolean) {
```

Comment:
Though, if we're using `copy` in order to change the only `val`, why not just construct a new version directly with a constructor?